datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -1054,6 +1061,44 @@ private void validateSdxResizeRequest(SdxCluster sdxCluster, String accountId, S
                 });
     }
 
+    private void validateDatalakeNotInBackupOrRestore(String sdxClusterName, String actorCrn) {
```

Comment:
Consider moving all of these helper files to the `SdxBackupRestoreService` just to keep things clean.

Reply:
Specifically we could just return a boolean over there and interpret that in this service, throwing an exception if necessary.

Reply:
@sxxgrc Moved this function to `SdxBackupRestoreService ` and used a boolean in the existing function `validateSdxResizeRequest`. Please let me know if it looks good. Thank you!!